test/libsolidity/StandardCompiler.cpp

```diff
@@ -41,6 +41,14 @@ namespace solidity::frontend::test
 namespace
 {
 
+set<string> getMemberNames(Json const& _json)
```

Comment:
Ideally instead of this we would use range-v3 to get the keys from the `items()` iterator, but haven't found a nice way yet.